MultiPointRasterGM.cpp

```diff
@@ -0,0 +1,43 @@
+//##############################################################################
+//
+//    FILENAME:          MultiPointRasterGM.cpp
+//
+//    CLASSIFICATION:    Unclassified
+//
+//    DESCRIPTION:
+//
+//    
+//
+//    LIMITATIONS:       None
+//
+//
+//    SOFTWARE HISTORY:
+//     Date          Author   Comment
+//     -----------   ------   -------
+//     31-Dec-2021  JPK    Initial version.
+//
+//    NOTES:
+//
+//##############################################################################
+
+#define CSM_LIBRARY
+
+#include "MultiPointRasterGM.h"
+
+namespace csm
+{
+	//*****************************************************************************
```

Comment:
Please, no tab characters.  Just use spaces to indent (though the other CSM code does not indent due to a `namespace`, so really no indent is needed here at all).

Reply:
done